.github/label-actions.yml

```diff
@@ -60,4 +60,17 @@ Needs Connection Logs:
 
     4. Get the log file and attach it to this issue. By default this will be in `%TMP%/LogFiles/Application/Functions/Host`.
 
-    **NOTE** Debug logging will include information such as Database and table names, if you do not wish to include this information you can either redact it from the logs before attaching or let us know and we will provide a way to send logs directly to us.
\ No newline at end of file
+    **NOTE** Debug logging will include information such as Database and table names, if you do not wish to include this information you can either redact it from the logs before attaching or let us know and we will provide a way to send logs directly to us.
+
+#actions for Needs More Info label
+Needs More Info:
+  comment: |
+    We need more info to debug your particular issue. If you could provide us with the following details, it would help us address the issue much faster.
+
+    1. Do you have any cutom settings for your function app. This could be a custom time-out defined specifically for your database server or optional configuration that can be cutomized for the app defined [here](https://learn.microsoft.com/en-us/azure/azure-functions/functions-bindings-azure-sql-trigger?tabs=isolated-process%2Cportal&pivots=programming-language-csharp#optional-configuration).
```

Comment:
```suggestion
    1. List any custom settings for your function app. This could be a custom time-out defined specifically for your database server or optional configuration that can be customized for the app defined [here](https://learn.microsoft.com/azure/azure-functions/functions-bindings-azure-sql-trigger?tabs=isolated-process%2Cportal&pivots=programming-language-csharp#optional-configuration).
```